src/app/components/challenge/challenge.component.html

```diff
@@ -1,5 +1,6 @@
 <app-header-static></app-header-static>
 <div class="mid-container">
+    <app-side-bar></app-side-bar>
```

Comment:
Please revert this change as this is not related to this PR.

Reply:
but sir then it will not look like the current version of evalai

Reply:
yes but this is not related to this PR. Sidebar will be added in a separate PR.